src/app/cluster/details/cluster/gatekeeper-config/gatekeeper-config-dialog/component.spec.ts

```diff
@@ -99,7 +99,7 @@ describe('GatekeeperConfigDialog', () => {
     });
 
     it('should call createGatekeeperConfig()', () => {
-      component.save();
+      component.getObservable().subscribe();
```

Comment:
Remove all those modified tests since they still do not make much sense and do not test what they should. They would only give a bad example for future reference to others.

Reply:
So you mean create new way to test it?
Or remove all the unit test